client/lib/domains/email-forwarding/reducer.js

```diff
@@ -118,6 +118,8 @@ function reducer( state, payload ) {
 		case UpgradesActionTypes.EMAIL_FORWARDING_DELETE_COMPLETED:
 			state = deleteTemporaryMailbox( state, action.domainName, action.mailbox );
 			break;
+		case UpgradesActionTypes.EMAIL_FORWARDING_RESEND_VERIFICATION_COMPLETED:
```

Comment:
Was wondering if it's better to leave it there, in case we do have something to add in the future.